pydal/_gae.py

```diff
@@ -83,3 +78,6 @@ def _validate(self, value):
                 return decimal.Decimal(value)
             raise TypeError("Property %s must be a Decimal or string."
                             % self._name)
```

Comment:
Any technical reason to do that?
With the changes on *adapters/\__init\__.py*, if something is wrong with the google adapters you will se the errors if your concerns are about this.